Dockerfile

```diff
@@ -10,6 +10,10 @@ COPY prepare-image.sh patch-image.sh /bin/
 RUN prepare-image.sh && \
   rm -f /bin/prepare-image.sh
 
+RUN mkdir -p /var/lib/ironic-inspector && \
+  sqlite3 /var/lib/ironic-inspector/ironic-inspector.db "pragma journal_mode=wal" && \
+  dnf remove -y sqlite
```

Comment:
I wonder if it will cause dnf to rebuild some caches already removed in prepare-image.

Reply:
it will rebuild some cache, it won't be as big as it was when running prepare-image.sh 
I can add a `dnf clean all` command at the end, or even split the one we have in prepare-image.sh in a new RUN
the final goal is to have the same prepare-image.sh scripts in both ironic-image and ironic-inspector-image, so the same will be done in ironic-image